users/serializers.py

```diff
@@ -239,11 +241,11 @@ def validate(self, data):
                 )
 
         username = data.get("username")
+        email = data.get("email")
```

Comment:
This is unused.

Reply:
it is used in the api call.

Reply:
Yeah, my bad. My IDE gave a warning so I just commented. Not sure why. It seems good now